readthedocs/projects/tasks/search.py

```diff
@@ -52,20 +52,20 @@ def fileify(version_pk, commit, build, search_ranking, search_ignore):
             search_ignore=search_ignore,
         )
     except Exception:
-        log.exception('Failed during ImportedFile creation')
+        log.warning("Failed during ImportedFile creation")
 
     # XXX: Don't access the sphinx domains table while we migrate the ID type
     # https://github.com/readthedocs/readthedocs.org/pull/9482.
     if not project.has_feature(Feature.DISABLE_SPHINX_DOMAINS):
         try:
             _create_intersphinx_data(version, commit, build)
         except Exception:
-            log.exception("Failed during SphinxDomain creation")
+            log.warning("Failed during SphinxDomain creation")
```

Comment:
This helped us to fix the problem with the pk passing the max value of an integer, I think is fine having all these as exception.